docs/getting_started/go/hello_world_in_go/index.md

```diff
@@ -18,61 +19,62 @@ image: /img/temporal-logo-twitter-card.png
 :::note Tutorial information
 
 - **Level:** ⭐ Temporal beginner
-- **Time:** ⏱️ ~10 minutes
+- **Time:** ⏱️ ~15 minutes
 - **Goals:** 🙌
   - Set up, build, and test a Temporal Application project from scratch using the [Go SDK](https://github.com/temporalio/sdk-go).
-  - Identify the four parts of a Temporal Application.
+  - Identify the four parts of a Temporal Workflow application.
   - Describe how the Temporal Server gets information to the Worker.
-  - Explain how to define Workflow Definitions with the Temporal Go SDK.
+  - Observe how Temporal recovers from failed Activities.
 
 :::
 
-
 ### Introduction
 
-Creating reliable applications is a difficult task.  [Temporal](https://temporal.io) lets you create fault-tolerant resilient applications using programming languages you already know, so you can build complex applications that execute successfully and recover from failures.
+Creating reliable applications is a difficult chore. [Temporal](https://temporal.io/) lets you create fault-tolerant, resilient applications using programming languages you already know, so you can build complex applications that execute reliably and recover from failures.
```

Comment:
This is extremely nitpicky, but "chore" in the first sentence sets a weird tone imo.